interactions/api/models/guild.py

```diff
@@ -655,6 +655,73 @@ async def modify_channel(
         )
         return Channel(**res, _client=self._client)
 
+    async def modify_member(
+        self,
+        member_id: int,
+        nick: Optional[str] = " ",  # can't be None because None would remove the nickname
+        roles: Optional[List[int]] = None,
+        mute: Optional[bool] = None,
+        deaf: Optional[bool] = None,
+        channel_id: Optional[
+            int
+        ] = 0,  # can't be None because None would kick the member from the channel
+        communication_disabled_until: Optional[
+            datetime.isoformat
+        ] = 0,  # can't be None because None would remove the timeout
+        reason: Optional[str] = None,
+    ) -> Member:
+        """
+        Modifies a member of the guild.
+
+        :param member_id: The id of the member to modify
+        :type member_id: int
+        :param nick?: The nickname of the member `None` removes the nickname
+        :type nick: Optional[str]
+        :param roles?: A list of all role ids the member has
+        :type roles: Optional[List[int]]
+        :param mute?: whether the user is muted in voice channels
+        :type mute: Optional[bool]
+        :param deaf?: whether the user is deafened in voice channels
+        :type deaf: Optional[bool]
+        :param channel_id?: id of channel to move user to (if they are connected to voice). `None` to disconnect the user from its channel
+        :type channel_id: Optional[int]
+        :param communication_disabled_until?: when the user's timeout will expire and the user will be able to communicate in the guild again (up to 28 days in the future). `None` removes the timeout
+        :type communication_disabled_until: Optional[datetime.isoformat]
+        :param reason?: The reason of the modifying
+        :type reason: Optional[str]
+        """
+        member = Member(**await self._client.get_member(guild_id=int(self.id), member_id=member_id))
+        _nick = member.nick if nick == " " else nick
```

Comment:
```suggestion
        _nick = member.nick if nick is None else nick
```

Good python practice would be doing `nick=""` in the method when written which strictly tells for the nickname to be removed. `nick=None` is a bad practice.